pkg/verify/verify.go

```diff
@@ -48,10 +63,10 @@ const maxSignatureSearch = 10
 // validReleaseDigest is a verification rule to filter clearly invalid digests.
 var validReleaseDigest = regexp.MustCompile(`^[a-zA-Z0-9:]+$`)
 
-// ReleaseVerifier implements a signature intersection operation on a provided release
+// releaseVerifier implements a signature intersection operation on a provided release
 // digest - all verifiers must have at least one valid signature attesting the release
 // digest. If any failure occurs the caller should assume the content is unverified.
-type ReleaseVerifier struct {
+type releaseVerifier struct {
```

Comment:
CVO, which will be reusing this package as well, needs access to field Store. Prior to any of my changes or refactoring to library-go ReleaseVerifier was also exported.

Reply:
If we want to re-use the interface, I'd suggest adding `Store` method to interface, directly accessing struct fields won't help here. And the current approach is causing nil-pointer exception when used in oc.